RecoTracker/IterativeTracking/python/iterativeTk_cff.py

```diff
@@ -43,6 +43,25 @@
     InitialStepPreSplitting +
     InitialStep +
     HighPtTripletStep +
+    DetachedQuadStep +
```

Comment:
On 3/20/16 4:16 AM, Marco Rovere wrote:
> In RecoTracker/IterativeTracking/python/iterativeTk_cff.py
> <https://github.com/cms-sw/cmssw/pull/13736#discussion_r56762564>:
> 
>> @@ -43,6 +43,25 @@
>>      InitialStepPreSplitting +
>>      InitialStep +
>>      HighPtTripletStep +
>> +    DetachedQuadStep +
> 
> Please refer to this talk
> <https://indico.cern.ch/event/281373/contribution/3/attachments/516584/712714/IterationSpeed4Nov13.pdf>
> to see why we swapped iterations.

Hi Marco,

I remember the changes made.
It's not fully clear why the trackingPhase1PU70 should be so pedantic to
try to keep what was there in SLHC release.

BTW, if we ever need to update to a higher PU (even PU70) before the end
of run 3,
I hope this trackingPhase1PU70  never gets revived and instead fades
away in git history.



> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/13736/files/26bfb299a20c0b2bdad1546baed59db4a816c7d0#r56762564>
>